docs/_sources/api.rst.txt

```diff
@@ -0,0 +1,57 @@
+.. _jurity API:
+
+Jurity Public API
+====================
+
+.. automodule:: jurity
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+Binary Classification Metrics
```

Comment:
Returns and Return Type are not consistently populated. Think it's the same for some of our other libraries so not a big issue, but wanted to make a note of this.